src/otx/v2/adapters/torch/lightning/modules/datasets/pipelines/sam_transforms.py

```diff
@@ -0,0 +1,114 @@
+"""SAM transfrom pipeline."""
```

Comment:
If this is only specific to sam then it should be located with rest of the sam code. But again, we haven't discussed the folder structure so this is just a comment for the future.